test/e2e/smoke-test/packages/local-path-storage/0.0.20/local-path-storage-test.sh

```diff
@@ -0,0 +1,63 @@
+#!/bin/bash
+
+# Copyright 2021 VMware Tanzu Community Edition contributors. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+# Checking package is installed or not
+tanzu package installed list | grep "local-path-storage.community.tanzu.vmware.com"
+exitcode=$?
+
+if [ "${exitcode}" == 1 ] 
+then
+  version=$(tanzu package available list local-path-storage.community.tanzu.vmware.com | tail -n 1 | awk '{print $2}')
+  tanzu package install local-path-storage --package-name local-path-storage.community.tanzu.vmware.com --version "${version}"
+fi
+
+echo "Wating for Pods to reach ready state ..."
+sleep 15s
+MY_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
+kubectl create -f "${MY_DIR}"/pvc.yaml
+kubectl create -f "${MY_DIR}"/pod.yaml
+
+echo "Creating pod to test local-path storage ..."
+sleep 30
+
+PVC_status="$(kubectl get pvc | grep "local-path-pvc" | awk '{print $2}')"
+
+Pod_status="$(kubectl get pod | grep "volume-test" | awk '{print $3}')"
```

Comment:
I agree that this method of getting PVC and Pod status works, and I'm a great fan of "it just works" way. I'm still wondering if we can just simplify it a bit but also add some small complexity -

One simplification is, avoid using `grep` since we know the exact name of the resource for both PVC and Pod -

```
kubectl get pod volume-test --no-headers
kubectl get pvc local-path-pvc --no-headers
```

For getting status, we can use `awk` or use `-o jsonpath=...` maybe

cc @aman556

Reply:
Thanks @karuppiah7890 makes sense I will edit this thing in script.